shared-libs/registration-utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@medic/registration-utils",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Should be `1.1.0` as you're adding a feature not fixing a bug.

Reply:
Done!